jenkins/e2e-image/Dockerfile

```diff
@@ -52,4 +52,7 @@ ADD ["e2e-runner.sh", \
     "/workspace/"]
 RUN ["chmod", "+x", "/workspace/get-kube.sh", "/workspace/upload-to-gcs.sh", "/workspace/sh2ju.sh"]
 
+ADD ["configure-image.sh", "/tmp"]
+RUN ["/tmp/configure-image.sh"]
```

Comment:
is there a reason not to just include the contents of this script directly in this Dockerfile?